packages/opentelemetry-metrics/src/MeterProvider.ts

```diff
@@ -54,4 +57,19 @@ export class MeterProvider implements api.MeterProvider {
 
     return this._meters.get(key)!;
   }
+
+  shutdown(cb: () => void = () => {}) {
+    this._shutdownAllMeters().then(() => {
+      setTimeout(cb, 0);
+    });
+  }
+
+  private async _shutdownAllMeters() {
+    if (this._config.exporter) {
+      this._config.exporter.shutdown();
```

Comment:
I don't think exporter shutdown calls are async. Regardless I'm not sure if it matters if this is blocking?

Reply:
I think the exporters don't do any batching so it should be ok to have the shutdown be sync. On the other hand, it may be worth adding to the API just to allow the flexibility in the future.

Reply:
Technically, the exporters could batch here: Say you have 1K unflushed spans, and the backend API supports 100 spans per batch call with a 500 ms limit between calls. -> shutdown would take 5 seconds.

I'm not sure it matters either. Perhaps something to potentially address in a future API design sprint.